<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterTourist</class>
 <widget class="QDialog" name="RegisterTourist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Register Tourist</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 8pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtFirstName">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 9pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 8pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
      <property name="text">
       <string>Middle Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtMiddleName">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 9pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 8pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtLastName">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 9pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 8pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
      <property name="text">
       <string>Passport No.</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="txtPassportNo">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 9pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 8pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
      <property name="text">
       <string>Contact No.</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="txtContactNo">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 9pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 5pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
      <property name="text">
       <string>Permanent Address</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 6pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
      <property name="text">
       <string>Choose Package</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTextEdit" name="txtPermanentAddress">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 9pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="cmbChoosePackage">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 9pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QTextEdit" name="txtPackageDescription">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 9pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 9pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
